pages/api/[entity_type_slug].js

```diff
@@ -55,6 +55,7 @@ async function get(req, res) {
         queries
     );
     const rawEntityType = await EntityType.find({ slug: entity_type_slug });
+    const isSingleType = (rawEntityType[0].entity_type_variant === 'singleton');
```

Comment:
'singleton' should be refactored as a constant.
```suggestion
    const isSingleType = (rawEntityType[0].entity_type_variant === 'singleton');
```